Library/Homebrew/system_config.rb

```diff
@@ -136,6 +149,7 @@ def dump_verbose_config(f = $stdout)
       f.puts "Perl: #{describe_perl}"
       f.puts "Python: #{describe_python}"
       f.puts "Ruby: #{describe_ruby}"
+      f.puts "System Ruby: #{describe_system_ruby}"
```

Comment:
This isn't the system Ruby, though, it's whatever Ruby is being used to execute Homebrew (which isn't necessarily the system one on non-OS X)